src/bindings/c/include/openvino/c/ov_prepostprocess.h

```diff
@@ -284,30 +285,47 @@ ov_preprocess_input_tensor_info_set_element_type(
  * @ingroup ov_prepostprocess_c_api
  * @param preprocess_input_tensor_info A pointer to the ov_preprocess_input_tensor_info_t.
  * @param colorFormat The enumerate of colorFormat
+ * @param sub_names_size The size of sub_names
+ * @param variadic params sub_names Optional list of sub-names assigned for each plane (e.g. "Y", "UV").
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_preprocess_input_tensor_info_set_color_format(
-                                ov_preprocess_input_tensor_info_t* preprocess_input_tensor_info,
-                                const ov_color_format_e colorFormat);
+ov_preprocess_input_tensor_info_set_color_format(ov_preprocess_input_tensor_info_t* preprocess_input_tensor_info,
+                                                 const ov_color_format_e colorFormat,
+                                                 const size_t sub_names_size,
```

Comment:
Honestly it is not a backward compatible change...((
Can we try to avoid backward incompatible changes?

Reply:
Frankly speaking, I also don't like such change too!
Previously, we had ignored the subname for colorFormat due to there is no any requirement.
But for remote tensor, especially for video decoder, there is nv12 format data, it required subname for each channel,  such as nv12 can attach "y" and "uv" subname.

To minimums the compatible issue, how to think below version?
```suggestion
                                                 const size_t sub_names_size = 0,
```

Reply:
It seems that C doesn't support default arguments, so maybe we can keep the old one and add a new one function to avoid compatible problem:

`ov_preprocess_input_tensor_info_set_color_format_with_subname(,,,)`